src/component/category-form/category-form.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import autoBind from '../../utils/';
+
+// this is the UI state. Everything but the store is the UI state from now on.
+const defaultState = {
+  title: '',
+};
```

Comment:
To fix the warning you are getting about `Controlled Components` in your console when you start typing in the input fields:

add to default state: `budget: 0` here and look for comment on line 35